### PR TITLE
Fix for Previous PR #149 

### PR DESCRIPTION
**Pull request by juro-ub**
Add FK

### CODE REVIEW

File: updates/23.sql [hunk 1]
```diff
@@ -16,3 +16,5 @@ INSERT INTO datenbank_mapping (Formular, Datenfeld, de_Beschriftung, Feldtyp, Ar
 VALUES ("einzelbeleg", "KritikSelektion", "Textkritische Anmerkung", "select", 0, "einzelbeleg", "KritikID", "selektion_kritik", "einzelbeleg", "Text-critical note", "Note critique sur le texte", "Nota cum textu critico");
 
 ALTER TABLE einzelbeleg ADD KritikID INT DEFAULT -1;
+
+ALTER TABLE einzelbeleg ADD FOREIGN KEY (KritikID) REFERENCES selektion_kritik(ID);
```
Comment (mtrojan-ub) [inline updates/23.sql]: All keys should also have distinct names

Comment (juro-ub) [reply updates/23.sql]: i have added a distinct name

File: updates/23.sql [hunk 2]
```diff
@@ -16,3 +16,5 @@ INSERT INTO datenbank_mapping (Formular, Datenfeld, de_Beschriftung, Feldtyp, Ar
 VALUES ("einzelbeleg", "KritikSelektion", "Textkritische Anmerkung", "select", 0, "einzelbeleg", "KritikID", "selektion_kritik", "einzelbeleg", "Text-critical note", "Note critique sur le texte", "Nota cum textu critico");
 
 ALTER TABLE einzelbeleg ADD KritikID INT DEFAULT -1;
+
+ALTER TABLE einzelbeleg ADD CONSTRAINT FK_EinzelbelegKritik_1 FOREIGN KEY (KritikID) REFERENCES selektion_kritik(ID);
```
Comment (mtrojan-ub) [inline updates/23.sql]: If you execute `SHOW CREATE TABLE einzelbeleg` you can see that all other existing foreign keys follow a naming scheme, e.g. `einzelbeleg_BearbeitungsstatusID`, `einzelbeleg_EditionID` and so on. Please change the name to `einzelbeleg_KritikID` according to the existing convention.

Comment (juro-ub) [reply updates/23.sql]: Fixed

**Pull request merged**